lib/pronto/formatter/pull_request_formatter.rb

```diff
@@ -3,6 +3,7 @@ module Formatter
     class PullRequestFormatter < GitFormatter
       def existing_comments(_, client, repo)
         sha = repo.head_commit_sha
+        puts "GETTING PULL COMMENTS"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.